memecity.engine/MemeEngine.cpp

```diff
@@ -22,6 +22,7 @@ namespace memecity::engine {
 			engine.timer.update();
 			engine.input_manager.update();
 			if (engine.timer.get_delta_time() >= 1.0f / 60.0f) {
+				engine.fps = 1.0f / engine.timer.get_delta_time();
```

Comment:
waarom hebben we geen methode `get_fps()`? nu doen we constant assignments, terwijl dat mss niet nodig is.